src/helpers/drm.ts

```diff
@@ -0,0 +1,57 @@
+import { DRMSystemConfiguration } from "../types";
+
+export const initFairPlayDRM = async (element: HTMLElement, fairPlayOptions: DRMSystemConfiguration) => {
+  const certificateUrl = fairPlayOptions.certificateUrl;
+  const licenseServerUrl = fairPlayOptions.licenseUrl;
+
+  const fairPlayCertificate = await loadFairPlayCertificate(certificateUrl);
+
+  element.addEventListener('encrypted', fairplayEncryptedCallback(fairPlayCertificate, licenseServerUrl));
+};
+
+const loadFairPlayCertificate = async (certificateUrl: string) => {
+  let response = await fetch(certificateUrl);
+  return response.arrayBuffer();
+}
+
+const fairplayEncryptedCallback = (fairPlayCertificate: ArrayBuffer, licenseServerUrl: string) => {
+  return async (event: MediaEncryptedEvent) => {
+    const video = event.target as HTMLVideoElement;
+    const initDataType = event.initDataType;
+
+    if (!video.mediaKeys) {
+      let access = await navigator.requestMediaKeySystemAccess("com.apple.fps", [{
+        initDataTypes: [initDataType],
+        videoCapabilities: [{ contentType: 'application/vnd.apple.mpegurl' }],
+      }]);
+
+      console.log(access);
```

Comment:
do we need this log? 
If so - let's use `debug` lib, here an example: https://github.com/Uscreen-video/video-player/blob/main/src/components/video-container/subtitles.ts#L7